### DISCUSSION TITLE
Collection filter by write permissions

### DISCUSSION THREAD

**Discussion by poborp** [appwrite/appwrite#1643] [Q&A] [answered]
Hi, thanks a lot for the project!!

Im playing with the database and filters, and I can't list documents filtering by the write access of my user. 

Its possible? something like this `filters[]=$permissions.write=user:6140fc5bbbd4d`

Cheers

**Comment by eldadfux** [comment, marked as answer]
Sorry for the late response, Hacktoberfest made our notifications to go wild. This is not possible, but you could store the relevant information in a traditional attribute and use it for sorting.
